demo/tests/test_end_point.py

```diff
@@ -919,6 +919,24 @@ def test_create_form_conditions_invalid_dup(self):
         # now that we allow multiple conditions for one field
         self.assertTrue(serializer.is_valid())
 
+    def test_create_form_conditions_null_name(self):
+        """
+        invalidate the condition using a condition name=None.
```

Comment:
Are we really "invalidating" the condition here ? I would have said transform / modify